### DISCUSSION TITLE
Removing obsolete HTML elements

### DISCUSSION THREAD

**Discussion by bsmth** [mdn/mdn-community#668] [Content] [answered]
We recently had a new issue opened about [`<menuitem>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menuitem) that asks for the page (and API page) to be removed: https://github.com/mdn/content/issues/32458. I took a look and would quote my reply here for convenience:

>We typically defer to BCD guidelines for removing obsolete data (to avoid duplicating policies and conversations around these rules). A feature or technology is a candidate for removal if:
>>a feature was implemented and has since been removed from all browsers dating back two or more years ago
>
>which matches this case, so I think removal of both of these pages is acceptable.

I discovered that we had this conversation before and it was decided to keep the pages in https://github.com/mdn/content/pull/24886. Unfortunately, the rationale for keeping the pages is not visible any more, so I'm opening a GitHub Discussion so there's a record.

The question is: Is it valuable to keep this information for historical purposes as an exception to our usual policies, or is it time to remove the content?

Looking forward to hearing what you think!

__Related issues and PRs:__

- [x] https://github.com/mdn/content/issues/32458
- [x] https://github.com/mdn/content/pull/24886
- [x] https://github.com/mdn/content/pull/24927
- Discussion https://github.com/orgs/mdn/discussions/620 about IE retirement.

**Comment by Akindin** [comment]
It will be good to have section of MDN for obsolete content that store different features for historical purposes, but I think this will require a lot of work and separate section from other content so as not to confuse someone who is looking for solution of their problem on MDN.

**Reply by bsmth** [reply]
For these cases, there's exceptions via Glossary entry, see:

* https://github.com/mdn/content/pull/35975

**Comment by Josh-Cena** [comment]
@estelle I wonder what the motivation for keeping the page is? (Since you commented in https://github.com/mdn/content/pull/24886 you might have some ideas.)

`<marquee>` is actually supported in browsers, but `<menuitem>` is not even valid markup.

**Reply by estelle** [reply]
My comment in #24866 was more "hesitation" than "opposition". If there's no BCD and it only ever worked in one browser anyhow, there is really no reason to keep the page.

**Reply by bsmth** [reply, marked as answer]
Thanks, both! If there's no opposition to removing the pages, I think we can accept a PR for deletion. I'm going to reply in the linked issue, so now's a good time in case you want to weigh in or if you've changed your mind.

**Reply by bsmth** [reply]
As mentioned above, there's exceptions we can make via Glossary entry, see:

https://github.com/mdn/content/pull/35975

Going to close as resolved for now!
